demo_classification_tf_for_database.py

```diff
@@ -0,0 +1,89 @@
+import tensorflow as tf
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+from os import listdir
+from os.path import isfile, join
+
+
+def sigmoid(x):
+  return 1 / (1 + np.exp(-x))
+
+def load_pb(path_to_pb):
+    with tf.io.gfile.GFile(path_to_pb, "rb") as f:
+        graph_def = tf.compat.v1.GraphDef()
+        graph_def.ParseFromString(f.read())
+        return graph_def
```

Comment:
Please unite common parts. I think that the best is to define a class with a method like `classify` that returns number of top predicted classes. Then just import this class here.